packages/wdio-browserstack-service/src/cli/cliUtils.ts

```diff
@@ -0,0 +1,357 @@
+import * as fs from 'node:fs'
+import { platform, arch, homedir } from 'node:os'
+import module from 'node:module'
+import path from 'node:path'
+import util from 'node:util'
+import url from 'node:url'
+import { exec } from 'node:child_process'
+import type { ZipFile } from 'yauzl'
+import type { IncomingMessage } from 'node:http'
+import yauzl from 'yauzl'
+import pkg from 'follow-redirects'
+const { https } = pkg
```

Comment:
As far as I know the package already uses `undici` for making requests, no need to add another package for it